proto/parca/debuginfo/v1alpha1/debuginfo.proto

```diff
@@ -58,3 +61,28 @@ message UploadResponse {
   // size is the number of bytes of the debug info
   uint64 size = 2;
 }
+
+// DownloadRequest upload debug info
+message DownloadRequest {
+  // build_id is a unique identifier for the debug data
+  string build_id = 1;
+}
+
+// DownloadRequest returns chunked data of the debuginfo.
+message DownloadResponse {
+  // data contains either the upload info metadata or the debug info
+  oneof data {
+    // info is the metadata for the debug info
+    DownloadInfo info = 1;
+
+    // chunk_data is the raw bytes of the debug info
+    bytes chunk_data = 2;
+  }
+}
+
+// DownloadInfo metadata for the debug data that is being downloaded.
+message DownloadInfo {
+  // from_public_debuginfod is true if the debug info is from public
+  // debuginfod server.
+  bool from_public_debuginfod = 1;
```

Comment:
How about introducing an enum for `source`? It would be more future-proof.
We already have an issue on it https://github.com/parca-dev/parca/issues/1258, if we change this now we don't need to change it in the future.

Reply:
Sounds good!